src/Components/Color/Color.jsx

```diff
@@ -40,7 +40,10 @@ export default function Color({ color, onDeleteColor }) {
           <button onClick={handleCancelDelete}>No</button>
         </div>
       ) : (
-        <button onClick={handleDeleteColor}>Delete</button>
+        <>
+          <button onClick={handleDeleteColor}>Delete</button>
+          <button onClick={handleDeleteColor}>Edit</button>
```

Comment:
You could add some classes for future styling. In my case, the code looks like this:
<button className="btn_edit" onClick={handleEditClick}>
  Edit
</button>